base/pkg/entry.jl

```diff
@@ -113,7 +111,7 @@ function installed(pkg::AbstractString)
     avail = Read.available(pkg)
     if Read.isinstalled(pkg)
         res = typemin(VersionNumber)
-        if ispath(joinpath(pkg,".git"))
+        if Read.isgitrepo(pkg)
```

Comment:
Note that `isinstalled` already checks for a case-sensitive `pkg`, so this change is somewhat redundant.  Doesn't hurt, I suppose.